src/Okta.Sdk/AppUserProfile.cs

```diff
@@ -3,10 +3,6 @@
 // Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
 // </copyright>
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Okta.Sdk
 {
     public class AppUserProfile : Resource
```

Comment:
Let's try also deleting this file and using errata to map it directly to `Resource`.